cmd/integration_link.go

```diff
@@ -253,6 +281,25 @@ List of available integrations:
 				errorQuit(errors.New("cannot define both destroy-on-stale and no-destroy-on-stale"))
 			}
 
+			allowReviewAppsFromForks := c.Bool("allow-review-apps-from-forks")
+			noAllowReviewAppsFromForks := c.Bool("no-allow-review-apps-from-forks")
+
+			if allowReviewAppsFromForks && noAllowReviewAppsFromForks {
+				errorQuit(errors.New("cannot define both allow-review-apps-from-forks and no-allow-review-apps-from-forks"))
+			}
+
+			awareOfSecurityRisks := c.Bool("aware-of-security-risks")
+
+			if allowReviewAppsFromForks && !awareOfSecurityRisks {
+				stillAllowed, err := isAutomaticReviewAppFromForksStillAllowedAwaringUserOfSecurityConcerns()
+				if err != nil {
+					errorQuit(err)
+				}
+				if err = c.Set("allow-review-apps-from-forks", strconv.FormatBool(stillAllowed)); err != nil {
```

Comment:
IIRC we try to avoid this one-liner form at scalingo

```suggestion
				err = c.Set("allow-review-apps-from-forks", strconv.FormatBool(stillAllowed))
				if err != nil {
```

Reply:
Oh, I didn't know that